internal/controllers/onboarding/tenant_reconciler_function.go

```
@@ -46,11 +47,12 @@ type FunctionBuilder struct {
 }
 
 type function struct {
-	logger         *slog.Logger
-	hubCache       controllers.HubCache
-	tenantsClient  privatev1.TenantsClient
-	hubsClient     privatev1.HubsClient
-	maskCalculator *masks.Calculator
+	logger          *slog.Logger
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(gofmt)